JWBestPracticeApps/FeedTableViewController/rename.swift

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/swift
+
+//
+//  main.swift
+//  XcodeProjectRenamer
+//
+//  Created by Marko Tadic on 8/1/16.
+//  Copyright © 2016 appculture. All rights reserved.
+//
+
+import Foundation
+
+class XcodeProjectRenamer: NSObject {
```

Comment:
I am confused by this class. What is it's purpose? Does it rename Xcode projects?
Or is this a simple set of utilities useful in this project?
If this is not being used, get rid of it. Otherwise, it needs a better name.

Reply:
Also if you are going to use it, note that this is copyrighted by AppCulture. Can we use this code?

>  Copyright © 2016 appculture

Reply:
Oh, didn't even notice this. We should not be using 3rd party code in our BPA.

Reply:
Whoops! I used this to rename the project recursively, throughout the project, IB, etc. It should not be here now that its job is done. Removed.